test-network-function/generic/suite.go

```diff
@@ -122,6 +128,87 @@ func createPartnerContainers(config *tnfConfig.TestConfiguration) map[tnfConfig.
 	return createContainers(config.PartnerContainers)
 }
 
+func pipeCmdToStd(cmd *exec.Cmd) {
+	cmd.Stdout = os.Stdout
+	cmd.Stderr = os.Stderr
+}
+
+func waitForPod() error {
+	for i := 0; i < 10; i++ {
+		checkPodStatusCmd := exec.Command("oc", "get", "pod", "node-master", "-o", "jsonpath='{.status.containerStatuses[0].ready}'")
+		var buffer bytes.Buffer
+		bufferWriter := bufio.NewWriter(&buffer)
+		checkPodStatusCmd.Stdout = bufferWriter
+		err := checkPodStatusCmd.Run()
+		if err != nil {
+			return err
+		}
+		cmdResult := buffer.String()
+		if cmdResult == "'true'" {
+			return nil
+		}
+		time.Sleep(time.Second * 2)
+	}
+	return errors.New("Passed timeout for pod to be ready")
+}
+
+func createPrivilagedPod() error {
+	ocArgs := []string{"apply", "-f", "../temp/privilaged-pod.yaml"}
+	createPrivilagedPodCmd := exec.Command("oc", ocArgs...)
+	pipeCmdToStd(createPrivilagedPodCmd)
+	err := createPrivilagedPodCmd.Run()
+	if err != nil {
+		return err
+	}
+
+	err = waitForPod()
+	if err != nil {
+		return err
+	}
+
+	return nil
+}
+
+func getPOTContainerId() (string, error) {
+	checkPodStatusCmd := exec.Command("oc", "get", "pod", "node-master", "-o", "jsonpath='{.status.containerStatuses[0].containerID}'")
```

Comment:
Same as above.  This should be a Test, not `exec.Command` per https://github.com/test-network-function/test-network-function/blob/main/CONTRIBUTING.md#test-implementation-guidelines